open-api/src/testFixtures/java/org/apache/iceberg/rest/RCKUtils.java

```diff
@@ -107,4 +116,18 @@ static void purgeCatalogTestEntries(RESTCatalog catalog) {
               catalog.dropNamespace(namespace);
             });
   }
+
+  public static int findFreePort() {
+    ServerSocket socket;
+    int port;
+    try {
+      socket = new ServerSocket(0);
+      port = socket.getLocalPort();
+      socket.close();
+    } catch (IOException e) {
+      throw new RuntimeException(e);
```

Comment:
Nit:  This could use try-with-resources and we should probably use UncheckedIOException rather than RE.

Reply:
Both done.

Reply:
Btw, since `RCKUtils` is being package protected again, and this findFreePort util is needed on the Spark integ side, I just moved this util out of `RCKUtils`.